shapeshift/web
# feat: send form defaults to amount denominated in crypto

## Description

This PR makes it so that the send form now by default has the amount denominated in crypto rater than fiat, #985

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)

closes #985

## Testing

Please outline all testing steps

1. Run local environment 
2. Connect a wallet with some crypto in it
3. Try sending the crypto and notice that the form now defaults to a crypto value
